### PR TITLE
I'm just add Lesson1

### PR DESCRIPTION
**Pull request by JackBenzzz**
Hi,

### CODE REVIEW

File: console-1/Lesson1.h [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#include <iostream>
+#pragma once
```
Comment (Modmitry) [inline console-1/Lesson1.h:1]: You should add includes after #pragma one raw.

File: console-1/Lesson1.h [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+	cout << "��� ������������ ������ ��������� ��� ������ ������������� � ���� ���������" << endl;
+	int a; cout << "������� ���������� �������� ���������, ������� ������������� �� ������: " << endl;
+	cin >> a;
+	int b; cout << "������� ���������� �������� ���������, ������� ������������� �� ������: " << endl;
+	cin >> b;
+
+
+	for (int w = 1; w <= a; w++)
+	{
```
Comment (Modmitry) [inline console-1/Lesson1.h:17]: It is general rule to use "i" and "j" as a name of indexes in cycles.

File: console-1/Lesson1.h [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+		for (int i = 1; i <= b; i++)
+		{
+			cout << "*";
+			if (i == b)
+			{
+				cout << "\n";
+				break;
+			}
```
Comment (Modmitry) [inline console-1/Lesson1.h]: We can remove this condition and add cout << "\n" just after internal cycle.

**Review by Modmitry — CHANGES_REQUESTED** [review]

**Comment by JackBenzzz** [comment]
Done, Dmitry!

**Comment by Modmitry** [comment]
Great job

**Review by Modmitry — APPROVED** [review]

**Pull request merged**